src/vs/editor/contrib/gotoError/browser/gotoError.ts

```diff
@@ -269,11 +269,11 @@ class MarkerNavigationWidget extends ZoneWidget {
 			// update frame color (only applied on 'show')
 			switch (marker.severity) {
 				case Severity.Error:
-					this.options.frameColor = '#ff5a5a';
+					this._container.parentElement.classList.add('marker-error-widget');
```

Comment:
Make sure to remove the warning class in this case, same as when you add a warning class remove the error class